driver/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ add_executable(MIOpenDriver
     dm_rnn.cpp
     dm_softmax.cpp
     dm_sum.cpp
+    dm_SGD.cpp
```

Comment:
Please follow the capitalization for the file names in other cases.
For example in this case, dm_sgd.cpp would be right.

Reply:
In the modnn, the opencl file name is `SGD.cl` so I thought we will capitalize all the `SGD` characters in files and code.
I will update all the file name from `...SGD...` to `...sgd...`. Thank you!